__tests__/send-tx.test.js

```diff
@@ -0,0 +1,121 @@
+import TestUtils from "./test-utils";
+
+describe("send-tx api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/send-tx")
+      .send({
+        outputs: [{ address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc", value: 1 }],
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
+  });
+
+  it("should accept value as string", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/send-tx")
+      .send({
+        outputs: [
+          { address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc", value: 1 },
+          { address: "WewDeXWyvHP7jJTs7tjLoQfoB72LLxJQqN", value: "1" },
+        ],
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeDefined();
```

Comment:
I feel we should check the success of the operation, shouldn't we?

Reply:
Done